service/src/main/kotlin/fi/espoo/evaka/reports/ManualDuplicationReportController.kt

```diff
@@ -23,7 +24,8 @@ class ManualDuplicationReportController(private val accessControl: AccessControl
     fun getPlacementCountReport(
         db: Database,
         user: AuthenticatedUser,
-        clock: EvakaClock
+        clock: EvakaClock,
+        @RequestParam("toggleDuplicatedCaseVisibility") toggleDuplicatedCaseVisibility: Boolean?
```

Comment:
The word "toggle" implies that the state of this is stored somewhere. The actual result of this doesn't seem to be very suitable to modeling by a boolean though 🤔 

Could the parameter be an enum with `DUPLICATED` and `NONDUPLICATED` or something?